internal/serverinstall/k8s.go

```diff
@@ -83,176 +79,145 @@ func (i *K8sInstaller) Install(
 		}
 	}
 
-	ui := opts.UI
 	log := opts.Log
+	ui := opts.UI
 
 	sg := ui.StepGroup()
 	defer sg.Wait()
 
-	s := sg.Add("Inspecting Kubernetes cluster...")
+	s := sg.Add("Getting Helm configs...")
 	defer func() { s.Abort() }()
-
-	clientset, err := i.newClient()
+	settings, err := helminstallutil.SettingsInit()
 	if err != nil {
-		ui.Output(err.Error(), terminal.WithErrorStyle())
 		return nil, err
 	}
+	s.Update("Helm settings retrieved")
+	s.Status(terminal.StatusOK)
+	s.Done()
```

Comment:
This is extra noise I think, we don't have to `Done` here. I'd group all of the `helm` related output into a single step group. Otherwise this feels noiser than it should:

```
brian@localghost:waypoint-webapp-go λ waypoint install -platform=kubernetes -accept-tos -k8s-server-image="localhost:5000/waypoint:dev"                               ±[●][main]
✓ Helm settings retrieved
✓ Helm action initialized
✓ Helm install created
✓ Helm chart located
✓ Helm chart loaded
```